src/Life/Main/Init.hs

```diff
@@ -35,8 +56,42 @@ lifeInit owner = do
         copyFile lifeFilePath lifePath
         createRepository owner (Repo "dotfiles")
   where
-    writeConf :: IO ()
-    writeConf = do
-        infoMessage "Writing .life configuration file"
-        let lifeConfig = singleFileConfig lifePath
+    createLifeFile :: IO ()
+    createLifeFile = do
+        infoMessage "Checking existence of some commonly used predefined files..."
+        (exist, noExist) <- scanConfig predefinedLifeConfig
+
+        unless (noExist == mempty) $ do
+            infoMessage "The following files and directories weren't found (so just ignoring them):"
+            skipMessage $ renderLifeConfiguration noExist
+
+        unless (exist == mempty) $ do
+            infoMessage "Found the following files and directories:"
+            successMessage $ renderLifeConfiguration exist
+
+        useDiscovered <- chooseYesNo "Would you like to add all discovered existing files and directories to .life configuration?"
+        let lifeConfig = singleFileConfig lifePath <> (if useDiscovered then exist else mempty)
+
+        infoMessage "Initializing global .life configuration file..."
         writeGlobalLife lifeConfig
+
+{- | Split given configuration into two:
+
+1. All files and directories which exist on machine.
+2. Other non-existing files and dirs.
+-}
+scanConfig :: LifeConfiguration -> IO (LifeConfiguration, LifeConfiguration)
```

Comment:
That's really nice that you prepared the interface already for any further configs (not only list of predefined files) 👍